lib/vast/Conversion/FromHL/ToHLBI.cpp

```diff
@@ -0,0 +1,434 @@
+            // case clang::Builtin::BI__builtin_coro_alloc:
+            // case clang::Builtin::BI__builtin_coro_begin:
+            // case clang::Builtin::BI__builtin_coro_end:
+            // case clang::Builtin::BI__builtin_coro_suspend:
+            // case clang::Builtin::BI__builtin_coro_align:
+
+            // case clang::Builtin::BI__builtin_coro_frame:
+            // case clang::Builtin::BI__builtin_coro_free:
+            // case clang::Builtin::BI__builtin_coro_size:
+            // case clang::Builtin::BI__builtin_dynamic_object_size:
+            // case clang::Builtin::BI__builtin_object_size:
+
+            // case clang::Builtin::BI__builtin_unreachable:
+
+            // case clang::Builtin::BImemcpy:
+            // case clang::Builtin::BI__builtin_memcpy:
+            // case clang::Builtin::BImempcpy:
+            // case clang::Builtin::BI__builtin_mempcpy:
+
+            // case clang::Builtin::BI__builtin_clrsb:
+            // case clang::Builtin::BI__builtin_clrsbl:
+            // case clang::Builtin::BI__builtin_clrsbll:
+
+            // case clang::Builtin::BI__builtin_ctzs:
+            // case clang::Builtin::BI__builtin_ctz:
+            // case clang::Builtin::BI__builtin_ctzl:
+            // case clang::Builtin::BI__builtin_ctzll:
+
+            // case clang::Builtin::BI__builtin_clzs:
+            // case clang::Builtin::BI__builtin_clz:
+            // case clang::Builtin::BI__builtin_clzl:
+            // case clang::Builtin::BI__builtin_clzll:
+
+            // case clang::Builtin::BI__builtin_ffs:
+            // case clang::Builtin::BI__builtin_ffsl:
+            // case clang::Builtin::BI__builtin_ffsll:
+
+            // case clang::Builtin::BI__builtin_parity:
+            // case clang::Builtin::BI__builtin_parityl:
+            // case clang::Builtin::BI__builtin_parityll:
+
+            // case clang::Builtin::BI__popcnt16:
+            // case clang::Builtin::BI__popcnt:
+            // case clang::Builtin::BI__popcnt64:
+            // case clang::Builtin::BI__builtin_popcount:
+            // case clang::Builtin::BI__builtin_popcountl:
+            // case clang::Builtin::BI__builtin_popcountll:
+
+            default:
+                return visitor.default_res();
+        }
+    }
+
+    template< typename call_op >
+    struct convert_builtin_operation : operation_conversion_pattern< call_op >
+    {
+        using base = operation_conversion_pattern< call_op >;
+        using base::base;
+
+        using adaptor_t = call_op::Adaptor;
+
+        logical_result
+        matchAndRewrite(call_op op, adaptor_t ops, conversion_rewriter &rew) const override {
+            auto rewriter = rewriter_visitor{ rew };
+            return visit_builtin_op(op.getOperation(), ops.getOperands(), rewriter);
+        }
+
+        static void legalize(conversion_target &trg) {
+            trg.addDynamicallyLegalOp< call_op >([](operation op) -> bool {
+                auto visitor = true_visitor{};
```

Comment:
simplify to single line

Reply:
add using to rename `true_visitor` to `is_visitable` to be more explicit what  it does